### ISSUE TITLE
(Gettext.Error) missing interpolation keys

### ISSUE THREAD

**Issue by nsi88** [elixir-lang/gettext#96]
Got:
```
 ** (Gettext.Error) missing interpolation keys: a
    (gettext) lib/gettext.ex:653: Gettext.handle_backend_result/1

```
when call:
`gettext("test %{a} %{a}", a: 1)`

For
`gettext("test %{a}", a: 1)`
everything is fine

**Comment by whatyouhide** [comment]
@nsi88 what version of Gettext are you running on? There was one issue with duplicate interpolation keys that was fixed recently (I'm on my phone so no link, but you can look through recent PRs if you want).

**Comment by whatyouhide** [comment]
Back at a computer, and this was definitely fixed in #92. We'll release a new gettext version that includes that change as soon as we'll fix a couple other things.
